### DISCUSSION TITLE
Allure report load but If I click on any item which redirects me to any other view in report, loading indicator appears and it is never stops loading

### DISCUSSION THREAD

**Discussion by KarMarcz** [allure-framework/allure2#2378] [Bug Reports] [answered]
Hi

From today We can`t read allure-report using https://marketplace.visualstudio.com/items?itemName=qameta.allure-azure-pipelines

Steps:
1. Enter Azure build with Allure report implemented
2. Go to the Allure Report Tab
3. The report is loaded
4. Click on any item in the Allure Report (for example Suites)

Expected Result:
Report is loaded

Actual Result:
![image](https://github.com/allure-framework/allure2/assets/39741797/96872b36-c904-4df6-8ef7-dc4b8cc3e5c8)

Note:
I did not get any errors on the console but I saw that the report not loading the rest of the report like in step 2

**Comment by epszaw** [comment, marked as answer]
Hey there! Try version 1.0.9 it should include fix for the problem.
